cnd/src/swap_protocols/herc20.rs

```diff
@@ -21,7 +21,19 @@ use crate::{ethereum::Bytes, timestamp::Timestamp};
 use blockchain_contracts::ethereum::rfc003::Erc20Htlc;
 pub use connector_impls::*;
 
-/// Resolves when said event has occured.
+/// Htlc ERC20 Token atomic swap protocol.
+
+/// Data required to create a swap that involves an ERC20 token.
```

Comment:
Not fully true, we are missing the other person's identity :)

Reply:
We don't require the other parties identity to create a swap, right?  We require that to 'finalize' a swap i.e., we learn it during the communication protocols.

Reply:
FTR I defined 'create a swap' in `db.rs` on the `CreatedSwap` struct.

Reply:
Depends on what you mean by "create a swap". If that means create the HTLCs / watch for them, then we do require the identites.

Reply:
I defined 'create a swap' in line with the identifiers we use `CreatedSwap` i.e., initiate the swap in cnd, post the swap on the REST API.

Reply:
oh, this might be a circular argument.  I thought we had this concept 'created' swap and 'finalized' swap.

Reply:
> I thought we had this concept 'created' swap and 'finalized' swap.

Unfortunately, I don't think that has settled yet as we can see from this discussion :see_no_evil:

Reply:
Ok, so I'm not sure what the action to carry out is after this thread of conversation, possible options:
- Remove all the comments on the various `CreatedSwap` structs?
- Rename all the `CreatedSwap` structs to something else?

I can do either of these but personally I couldn't come up with anything better than whats here, a not-perfect name with documentation explaining what the name means.

FTR, this patch also adds (`db.rs`)
```
/// Data required to create a swap.
///
/// 'create' a swap is defined as the process of initiating a swap within `cnd`.
/// The data required to do so is assumed to have been negotiated between the
/// two parties prior to each creating the swap.
#[derive(Debug, Clone)]
pub struct CreatedSwap<A, B> {
```